### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by mbarbetti**
* added the link to my personal academic website ([mbarbetti.github.io](https://mbarbetti.github.io))
* added the link to the website for a workshop named "Beyond Vision: Physics meets AI" ([BVPAI 2023](https://physicsmeetsai.github.io/beyond-vision)) held within ICIAP 2023

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -163,10 +165,11 @@ Medical Robotics Junior Faculty Forum (ISMR: <a href="https://junior-forum-ismr.
 
 ## Table Of Contents
 
-  * [User community](#user-community)
```
Comment (pourmand1376) [inline README.md:166]: Is there any reason why the table of contents is changed?

File: README.md [hunk 2]
```diff
@@ -188,6 +191,35 @@ Medical Robotics Junior Faculty Forum (ISMR: <a href="https://junior-forum-ismr.
...
+  - [Features](#features)
+    - [Publications](#publications)
+    - [Collections](#collections)
+    - [Layouts](#layouts)
+      - [The iconic style of Distill](#the-iconic-style-of-distill)
+      - [Full support for math \& code](#full-support-for-math--code)
+      - [Photos](#photos)
+    - [Other features](#other-features)
+      - [GitHub's repositories and user stats](#githubs-repositories-and-user-stats)
+      - [Theming](#theming)
```
Comment (pourmand1376) [inline README.md:216]: Are these accidental?

Comment (mbarbetti) [reply README.md:216]: Uh, I think so. Actually, I didn't realize to have changed these rows

File: README.md (hunk 1 above)
Comment (mbarbetti) [reply README.md:166]: These modifications were accidental and, actually, I don't know how it happened... sorry!

**Pull request merged**